CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
         - Only trigger one refresh going Back to list view. #3476
         - Fix checked order of updates in dashboard export.
         - Fix unable to edit user with verified landline #3295
+        - Fix ordering of dropdown lists in extra questions #2511
```

Comment:
This is a public file, so any issue numbers mentioned can only be public issue IDs (if there isn't a public bug issue (no reason there couldn't have been in this case, but doesn't matter now), then generally we put the PR number here instead so the line can be followed back to the relevant change).